apis/v1alpha2/policy_types.go

```diff
@@ -120,3 +122,81 @@ const (
 	// policy is attached to an invalid target resource.
 	PolicyReasonTargetNotFound PolicyConditionReason = "TargetNotFound"
 )
+
```

Comment:
Could you update L79 in`PolicyTargetReferenceWithSectionName` : "a `ResolvedRefs` or similar Condition in the Policy's status." to what it should be, either `TargetNotFound` or add a new condition `TargetSectionNotFound`?

Reply:
Good catch! I think this might actually be best for a follow up PR because we probably also want to have `TargetSectionNotFound` like you're suggesting, but I think it would be cleaner to add a new condition like that in a separate PR.